# An Array of radar obstacles in DuDrive

Header header

DriveRadarObs[]  obstacles

